_infra/helm/frontstage/templates/deployment.yaml

```diff
@@ -193,9 +193,10 @@ spec:
             {{- else }}
             value: "http://$(SURVEY_SERVICE_HOST):$(SURVEY_SERVICE_PORT)"
             {{- end }}
-          - name: ZIPKIN_DSN
-            value: "http://$(ZIPKIN_SERVICE_HOST):$(ZIPKIN_SERVICE_PORT)/api/v1/spans"
-          - name: ZIPKIN_SAMPLE_RATE
+          - name: SURVEY_USERNAME
```

Comment:
I'm pretty sure these aren't needed anymore.  Do a search to see if it's used anywhere